### PR TITLE
feat: LC-481 scorm student view

### PR DESCRIPTION
**Pull request by typeWolffo**
## Jira issue(s)

https://selleolabs.atlassian.net/browse/LC-481

## Overview
- added scorm lesson view

### Notes
- scorm iframe url is is temporarily hardcoded just for demo purposes

### CODE REVIEW

File: apps/api/src/scorm/services/scorm.service.ts [hunk 1]
```diff
@@ -113,15 +117,161 @@ export class ScormService {
...
+    if (relativePath.startsWith("/")) {
+      return relativePath.slice(1);
+    }
+
+    const baseDir = path.dirname(basePath);
+    if (relativePath.startsWith("../")) {
+      const parts = baseDir.split("/");
+      const relParts = relativePath.split("/");
+
+      let upCount = 0;
```
Comment (typeWolffo) [inline apps/api/src/scorm/services/scorm.service.ts:250]: ✂️

File: apps/api/package.json [hunk 2]
```diff
@@ -64,6 +65,7 @@
     "drizzle-orm": "0.31.2",
     "drizzle-typebox": "0.1.1",
     "faker": "link:@types/@faker-js/faker",
+    "jsdom": "^24.1.1",
```
Comment (typeWolffo) [inline apps/api/package.json:68]:
```suggestion
    "jsdom": "24.1.1",
```

File: apps/web/app/modules/Courses/CourseView/CourseView.page.tsx [hunk 3]
```diff
@@ -59,8 +64,20 @@ export default function CoursesViewPage() {
         </BreadcrumbItem>
       </BreadcrumbList>
       <div className="flex flex-col md:flex-row h-full gap-6">
-        <CourseViewMainCard {...course} />
-        <LessonsList lessons={course.chapters} isEnrolled={course.enrolled || isAdmin} />
+        {course.isScorm ? (
+          <iframe
+            title="Playing"
+            src="https://api.lms.localhost/api/scorm/dd284185-2e3b-4a06-8736-403a421a32a5/content?path=Playing/Playing.html"
```
Comment (typeWolffo) [inline apps/web/app/modules/Courses/CourseView/CourseView.page.tsx:70]: iframe url is is temporarily hardcoded just for demo purposes

**Review by wielopolski — APPROVED** [review]
LG 🍻

**Pull request merged**